DocumentationTests/DocumentationTests/DocumentationTests/GloballyUsedVariables.swift

```diff
@@ -0,0 +1,277 @@
+import StreamVideo
+import StreamVideoSwiftUI
+import SwiftUI
+
+var apiKey = ""
+var user = User(id: "")
+var token = UserToken(stringLiteral: "")
+var callId = ""
+var tokenProvider: UserTokenProvider = { _ in }
+var streamVideo = StreamVideo(apiKey: apiKey, user: user, token: token)
+var streamVideoUI = StreamVideoUI(streamVideo: streamVideo)
+var call = streamVideo.call(callType: "default", callId: UUID().uuidString)
+var utils = Utils()
+@MainActor var viewModel = CallViewModel()
+@MainActor var viewFactory = DefaultViewFactory.shared
+var availableFrame = CGRect.zero
+var availableSize = CGSize.zero
+var videoSize = CGSize.zero
+var participant = CallParticipant(
+    id: "",
+    userId: "",
+    roles: [],
+    name: "",
+    profileImageURL: nil,
+    trackLookupPrefix: nil,
+    hasVideo: false,
+    hasAudio: false,
+    isScreenSharing: false,
+    showTrack: false,
+    isDominantSpeaker: false,
+    sessionId: "",
+    connectionQuality: .unknown,
+    joinedAt: .init(),
+    audioLevel: 0,
+    audioLevels: [],
+    pin: nil
+)
+var contentMode = UIView.ContentMode.scaleAspectFit
+var id = ""
+var customData = [String: RawJSON]()
+var ratio: CGFloat = 0
+var onChangeTrackVisibility: @MainActor(CallParticipant, Bool) -> Void = { _, _ in }
+var orientation: UIInterfaceOrientation = .unknown
+var localParticipant = participant
+var reader: GeometryProxy!
+var participants = [participant]
+var imageURL: URL!
+var members: [MemberRequest] = []
+
+func container(_ content: () -> Void) {}
+func asyncContainer(_ content: () async throws -> Void) {}
+func viewContainer(@ViewBuilder _ content: () -> some View) {}
+func classContainer<V: AnyObject>(_ content: (V) -> Void) {}
+
+struct UserCredentials {
+    let user: User
+    let token: UserToken
+}
+
+extension UserCredentials {
+    static let demoUser = UserCredentials(
+        user: User(
+            id: "testuser",
+            name: "Test User",
+            imageURL: URL(string: "https://vignette.wikia.nocookie.net/starwars/images/2/20/LukeTLJ.jpg")!,
+            customData: [:]
+        ),
+        token: UserToken(rawValue: "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdHJlYW0tdmlkZW8tZ29AdjAuMS4wIiwic3ViIjoidXNlci90ZXN0dXNlciIsImlhdCI6MTY2NjY5ODczMSwidXNlcl9pZCI6InRlc3R1c2VyIn0.h4lnaF6OFYaNPjeK8uFkKirR5kHtj1vAKuipq3A5nM0")
```

Comment:
do we need the token? Since the API key is also blank, I believe we can make this an empty string, too?